GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_matched.yaml

```diff
@@ -0,0 +1,12 @@
+subclass_name: dc2_matched_table.DC2MatchedTable
+table_dir: /global/projecta/projectdirs/lsst/groups/SSim/DC2/
+table_filename_template: matched_ids_run{}_{}_{{}}.fits.gz
+data_release: dr1b_dpdd
+version: '2.1i'
+truth_version: 'cosmoDC2_v1.1.4'
+object_id: 'objectId'
+truth_id: 'truthId'
+match_flag: 'is_matched'
+is_star: 'is_star'
+creator: 'Javier Sanchez'
+description: 'This is the table of matches between the object catalog and the extragalactic catalog'
```

Comment:
should we specify what kind of matching was applied, and a link to the code which did the matching. Provenance seems not easy to track as it stands. Maybe the matching code "as run" could make it to a separate directory in GCRCatalogs?

Reply:
This is the code used to match the inputs and outputs: https://github.com/LSSTDESC/CatalogMatcher/blob/master/CatalogMatcher/match.py#L44

In particular I used `npix=3` and `max_deltamag=1.0`

Reply:
The complete script lives [here](https://gist.github.com/fjaviersanchez/787bb5cd6b598226174e1cd9661465ca)

Reply:
thanks @fjaviersanchez ! Ideally for provenance we would have a tag of the script in the github repo you are using. Maybe a static reference to a perennial gist is also ok at this point, but I like less the idea.

Reply:
I added these locations to the doc strings in the code.

Reply:
I can put that code in the DC2-production repo

Reply:
I am not sure what is best..... Other opinions welcome : @heather999 @yymao @wmwv

Reply:
I think the code belongs in a LSSTDESC repo which is tagged.  Unfortunately, we do not do a good job of tagging DC2-production, but I'm open to use that repo for this purpose... and then we should take a quick look at the outstanding PRs and make a DC2-production release tag.